patches/chrome-browser-resources-extensions-BUILD.gn.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
-index cfed148d68be8d0880823b391b55936f942c22cc..2b2989f4ce45b6b74d25d3537865d7d09c54c19e 100644
+index cfed148d68be8d0880823b391b55936f942c22cc..13eaf87e24b8203739cf5c51d4d0356f5052384e 100644
 --- a/chrome/browser/resources/extensions/BUILD.gn
 +++ b/chrome/browser/resources/extensions/BUILD.gn
-@@ -57,6 +57,7 @@ generate_grd("build_grd") {
+@@ -30,6 +30,7 @@ if (optimize_webui) {
+       "../../../../ui/webui/resources:preprocess",
+     ]
+     excludes = [ "chrome://resources/js/cr.m.js" ]
++    deps += brave_deps_chrome_browser_resources_extensions_build
```

Comment:
do we still need the other patch in this file? https://github.com/brave/brave-core/pull/7599/files#diff-5606dbee6795afa01ac32eebfe4d4cfc97f19d928fc34fac419a49599530a6adR17

Reply:
I believe we do as this should be used for our settings customizations
cc: @petemill

Reply:
@bridiver I believe we do as they are different sets of files used at different steps of the build

Reply:
@bsclifton same applies to both settings and extensions. They both need 2 patches as far as I can tell so far.